pkg/distributors/contracts/MultiDistributor.sol

```diff
@@ -177,83 +182,87 @@ contract MultiDistributor is IMultiDistributor, ReentrancyGuard, MultiDistributo
     }
 
     /**
-     * @dev Creates a new distribution
-     * @param stakingToken The staking token that will be eligible for this distribution
+     * @dev Creates a new distribution channel
+     * @param stakingToken The staking token that will be eligible for this distribution channel
      * @param distributionToken The token to be distributed to users
      * @param duration The duration over which each distribution is spread
      */
     function createDistribution(
```

Comment:
Shouldn't this be renamed? If distributions last for some time after they are funded and run through a channel, then what this creates is the channel.